packages/story-utils/src/generateSnapshots.ts

```diff
@@ -1,7 +1,9 @@
-import { composeStories, composeStory } from '@storybook/testing-react';
-import type { StoryFile } from '@storybook/testing-react/dist/types';
-import { render, RenderResult } from '@testing-library/react';
-import { createElement } from 'react';
+import type {composeStory} from '@storybook/testing-react';
+import {composeStories} from '@storybook/testing-react';
```

Comment:
This is interesting.

Note that the lint rules split `composeStories` and `composeStory` into different imports, based on how they're used (we only use `composeStory` for its types).

```diff
-import { composeStories, composeStory } from '@storybook/testing-react';
+import type {composeStory} from '@storybook/testing-react';
+import {composeStories} from '@storybook/testing-react';
```

Pretty cool. Nice find on the `@typescript-eslint/consistent-type-imports` rule, @jeremiah-clothier